bundles/binding/org.openhab.binding.zwave/database/aeon/zw096.xml

```diff
@@ -205,7 +204,6 @@
             <Type>byte</Type>
             <Default>505050</Default>
```

Comment:
> On 11 Jul 2015, at 15:05, Jim <notifications@github.com> wrote:
> 
> So... 323232 equates to decimal 3289650 and I will make that the default value.
> and .. 636363 equates to decimal 6513507 and that will be the max.
> 
> As noted previously, this smells bad.
> 
I agree - it’s not the best, but there’s not much we can do about this, so I think it’s ok...